aocd/examples.py

```diff
@@ -16,7 +19,11 @@
 from aocd.utils import get_plugins
 
 
-log = logging.getLogger(__name__)
+log: logging.Logger = logging.getLogger(__name__)
```

Comment:
The `log` objects in each module should *probably* be private, so `_log`. The only reason I annotated them is because they are part of the public API.